hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1370,17 +1370,10 @@ public boolean addAcl(OzoneObj obj, OzoneAcl acl) throws IOException {
       validateBucket(volume, bucket);
       String objectKey = metadataManager.getOzoneKey(volume, bucket, keyName);
       OmKeyInfo keyInfo = metadataManager.getKeyTable().get(objectKey);
-      Table keyTable;
       if (keyInfo == null) {
```

Comment:
Since we don't allow multiple clients write to the the key concurrently. So there should be only one entry in the open key table for a key being written. 

Instead of disallowing acl operations on open key, shall we use prefix seek (without clientID) in open key table to find the match entries here?

Reply:
Sure we can do that? What is the use case for setting acl for an openKey which is not committed to OM. Just want to understand the reason behind it.

Reply:
I don't have specific use case for setting acl for uncommitted key as it is only visible to the client with lease. Based on that, I think it is OK to skip check the openKeyTable for acl.